packages/client/components/TeamPrompt/Recurrence/TimeLeftBadge.tsx

```diff
@@ -25,7 +26,7 @@ export const TimeLeftBadge = (props: Props) => {
       <TeamPromptBadge onMouseEnter={openTooltip} onMouseLeave={closeTooltip} ref={originRef}>
```

Comment:
+1 badges shouldn't show the caret when hovered. maybe `user-select: none` or change the cursor?

Reply:
YES, I don't see anything better than cursor `help` which isn't ideal. I'd go with `user-select: none`